.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Describe the content of the PR and links to related issues, bugs of features.
```

Comment:
I would suggest expanding this out a bit more, including some (but not all) of the content from the [fv3net template.](https://github.com/VulcanClimateModeling/fv3net/blob/master/.github/pull_request_template.md).

A good set of headings might be "Code changes" with a bullet, and "requirement changes" with a bullet. We don't have much public API or multiple packages with inter-dependencies the way fv3net does, so the stress on internal vs public changes isn't as important. This might change later and we could re-visit it.

Reply:
Thanks for sharing that, helpful!